app/forms/AssociatedCompaniesFormProvider.scala

```diff
@@ -39,7 +39,7 @@ class AssociatedCompaniesFormProvider @Inject() extends Mappings {
           optional(
             int(
               "associatedCompaniesCount.error.required",
-              "associatedCompaniesCount.error.wholeNumber",
+              "error.greaterThan99",
```

Comment:
I don't think this should change - it relates to the error shown when a user enters decimal points for example, if I enter `1.3` I should be told that I need to enter a whole number which is what this error was doing before - now it will tell me it should be less than 99...